CMakeLists.txt

```diff
@@ -18,6 +18,12 @@ set(SMQTK_VERSION_MAJOR ${CMAKE_MATCH_1})
 set(SMQTK_VERSION_MINOR ${CMAKE_MATCH_3})
 set(SMQTK_VERSION_PATCH ${CMAKE_MATCH_5})
 
+###
+# Options
+#
+option(SMQTK_BUILD_EXAMPLES "Enable building examples." ON)
+option(SMQTK_INSTALL_SETUP_SCRIPT "Enable building examples." ON)
+mark_as_advanced(SMQTK_INSTALL_SETUP_SCRIPT)
```

Comment:
Is this defaulted to anything somewhere? Maybe we should just not install the setup script when we're on windows?